resources/lib/syncEpisodes.py

```diff
@@ -578,10 +575,10 @@ def __compareShows(self, shows_col1, shows_col2, rating=False, restrict=False):
 
                 if show_col2:
                     show = {'title': show_col1['title'], 'ids': {}, 'year': show_col1['year']}
-                    if 'tvdb' in show_col1['ids']:
-                        show['ids'] = {'tvdb': show_col1['ids']['tvdb']}
-                    if 'imdb' in show_col2 and show_col2['imdb']:
-                        show['ids']['imdb'] = show_col2['imdb']
+                    if show_col1['ids']:
+                        show['ids'].update(show_col1['ids'])
+                    if show_col2['ids']:
+                        show['ids'].update(show_col2['ids'])
```

Comment:
It will add missing keys and update existing ones. Should not be relevant as we already decided that col1 and col2 are matching shows, shouldn't it? That's pretty much what was being done before where tvdb was picked from col1 and imdb from col2. Why not all?